### PR TITLE
fix color picker (signal handler + alpha channel)

### PR DESCRIPTION
**Pull request by nothingTVatYT**
This fixes the signal handling problem on Linux which closes the dialog before the color under the mouse cursor is accepted and the alpha channel of the picked color was always 0.

### CODE REVIEW

**Comment by nothingTVatYT** [comment]
Fixes #1694

File: Source/Editor/GUI/Dialogs/ColorPickerDialog.cs [hunk 1]
```diff
@@ -218,6 +218,7 @@ public ColorPickerDialog(Color initialValue, ColorValueBox.ColorPickerEvent colo
 
         private void OnColorPicked(Color32 colorPicked)
         {
+            Editor.Log("OnColorPicked " + colorPicked + ", _active eye dropper=" + _activeEyedropper);
```
Comment (mafiesto4) [inline Source/Editor/GUI/Dialogs/ColorPickerDialog.cs:221]: This could be removed

File: Source/Editor/GUI/Dialogs/ColorPickerDialog.cs [hunk 2]
```diff
@@ -319,7 +320,7 @@ public override void Draw()
         protected override void OnShow()
         {
             // Auto cancel on lost focus
-            ((WindowRootControl)Root).Window.LostFocus += OnCancel;
+            //((WindowRootControl)Root).Window.LostFocus += OnCancel;
```
Comment (mafiesto4) [inline Source/Editor/GUI/Dialogs/ColorPickerDialog.cs:323]: Wrap it around `#ifdef !PLATFORM_LINUX`/`#endif` instead.

**Review by mafiesto4 — CHANGES_REQUESTED** [review]

File: Source/Editor/GUI/Dialogs/ColorPickerDialog.cs (hunk 1 above)
Comment (nothingTVatYT) [reply Source/Editor/GUI/Dialogs/ColorPickerDialog.cs]: Oh, missed that one.

File: Source/Editor/GUI/Dialogs/ColorPickerDialog.cs (hunk 2 above)
Comment (nothingTVatYT) [reply Source/Editor/GUI/Dialogs/ColorPickerDialog.cs]: Done.

**Review by mafiesto4 — APPROVED** [review]

**Pull request merged**